tensorrt_llm/executor/ipc.py

```diff
@@ -26,18 +26,42 @@ class ZeroMqQueue:
         zmq.PUSH: "PUSH",
     }
 
+    # Base serializable types and their handlers
+    BASE_SERIALIZABLE_TYPES = {
```

Comment:
Can you add more descriptions here to guide some developers in updating this? They may introduce a new type, break the tests, and when they finally switch here and the comments should help them to briefly know the background.